docs/faq.md

```diff
@@ -0,0 +1,250 @@
+# The Flax FAQ
```

Comment:
Can you please remove the FAQ from this pull request? That should be in a separate commit.

You can do this with `git rebase -i` to remove the FAQ commit from this branch's history, then `git push --force`.

Reply:
Done!